composer.json

```diff
@@ -34,10 +34,18 @@
         ]
     },
     "scripts": {
-        "dev-site": [
-            "@php docroot/core/scripts/dev-site.php install",
-            "@php bin/drush en -y jsonapi admin_ui_support",
-            "@php docroot/core/scripts/dev-site.php start"
+        "install": [
+            "php docroot/core/scripts/drupal install standard --no-interaction",
+            "vendor/bin/drush en -y jsonapi admin_ui_support"
+        ],
+        "start": [
+            "cd docroot && php core/scripts/drupal server"
+        ],
+        "devify": [
+            "rm -rf docroot/vfancy",
```

Comment:
While reviewing we realized that this doesn't allow Drupal to use it, which is fine. Maybe we should just check the folder out somewhere else.